lib/routes.js

```diff
@@ -108,12 +108,16 @@ module.exports = (server) => {
                     tutorial = Tutorials[language].default;
                 }
 
+                const menus = Tutorials[language].all;
+                const firstItem = 'getting-started';
+                const item = menus.filter(({ slug }) => slug === firstItem);
```

Comment:
When looking for 1 item in an array it is faster to user `Array.find` it does basically the same thing but stops looping when it finds the first match. ✨
In this case it won't really matter since we loop over 10 items or so, so you will never notice it :D

Reply:
hmmm, that's fine! I'll fix this 😄